content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,120 +11,79 @@ weight: 10
 
 <!-- overview -->
 
-On-disk files in a Container are ephemeral, which presents some problems for
-non-trivial applications when running in Containers.  First, when a Container
-crashes, kubelet will restart it, but the files will be lost - the
-Container starts with a clean state.  Second, when running Containers together
-in a `Pod` it is often necessary to share files between those Containers.  The
-Kubernetes `Volume` abstraction solves both of these problems.
-
-Familiarity with [Pods](/docs/concepts/workloads/pods/pod/) is suggested.
+On-disk files in a container are ephemeral, which presents some problems for
+non-trivial applications when running in containers. One problem
+is the loss of files when a container crashes. The kubelet restarts the container
+but with a clean state. A second problem occurs when sharing files
+between containers running together in a pod.
+The Kubernetes {{< glossary_tooltip text="volume" term_id="volume" >}} abstraction
+solves both of these problems.
+Familiarity with [Pods](/docs/concepts/workloads/pods/) is suggested.
 
 <!-- body -->
 
 ## Background
 
-Docker also has a concept of
+Docker has a concept of
 [volumes](https://docs.docker.com/storage/), though it is
-somewhat looser and less managed.  In Docker, a volume is simply a directory on
-disk or in another Container.  Lifetimes are not managed and until very
-recently there were only local-disk-backed volumes.  Docker now provides volume
-drivers, but the functionality is very limited for now (e.g. as of Docker 1.7
-only one volume driver is allowed per Container and there is no way to pass
-parameters to volumes).
-
-A Kubernetes volume, on the other hand, has an explicit lifetime - the same as
-the Pod that encloses it.  Consequently, a volume outlives any Containers that run
-within the Pod, and data is preserved across Container restarts. Of course, when a
-Pod ceases to exist, the volume will cease to exist, too.  Perhaps more
-importantly than this, Kubernetes supports many types of volumes, and a Pod can
-use any number of them simultaneously.
+somewhat looser and less managed. A Docker volume is a directory on
+disk or in another container. Docker provides volume
+drivers, but the functionality is somewhat limited.
+
+Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod" text="Pod" >}}
+can use any number of volume types simultaneously. The lifetime of a Kubernetes volume
```

Comment:
The note about volume lifetime is not correct. Ephemeral volume types have a lifetime of a pod, but persistent volumes have a lifetime beyond the pod.

Reply:
Hi @msau42 .
Thanks for the feedback. Initially, I created this PR to clean up the formatting and styling
of the page content. I was also experimenting with adapting some new docs style guidelines.
I went a bit further by modifying some of the text (grammar, readability), but
did not delve too deeply into the content. I'll look again at the page and see if I can make more content changes
otherwise, I can log an issue to review the page in more detail. How does that sound?

Reply:
Let me see if I can updated correctly.

Reply:
Do you think it is still relevant to include the paragraph about Docker volumes (as a comparison)?

Reply:
I think leaving content updates to a future pr is fine. I think the Docker volumes comparison may need to be updated as well. The biggest difference I think that's relevant is Docker volumes write data to the host and do not persist if the pod is restarted on a different node.